### ISSUE TITLE
Ä° and Ä« not detected as mojibake

### ISSUE THREAD

**Issue by robstradling** [rspeer/python-ftfy#192]
Hi @rspeer.  Many thanks for creating and maintaining FTFY!  We're using it at Sectigo to help prevent mojibake from finding its way into string fields in the digital certificates that we issue.  We've noticed a couple of mojibake sequences that FTFY doesn't currently detect and fix:

Desired behaviour:
```
$ echo "Ä°stanbul" | iconv -t WINDOWS-1252
İstanbul
$ echo "RÄ«ga" | iconv -t WINDOWS-1252
Rīga
```

Current FTFY behaviour:
```
$ echo "Ä°stanbul" | ftfy
Ä°stanbul
$ echo "RÄ«ga" | ftfy
RÄ«ga
```

Would it be possible to make FTFY handle these cases?

**Comment by rspeer** [comment]
I found a fairly natural place to add this to the heuristic. Thanks!

**Comment by rspeer** [comment]
Released in 6.3.
